### ISSUE TITLE
Cannot save abbreviation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#185]
```
What steps will reproduce the problem?
Problem:Try to add new phrase and set abbreviation
OR modify existing abbreviation
BUT there is no problem with setting up hotkeys
1. open Main window
2. Add New Phrase/call it say 'a'/ Abbreviation - Set/ say 'new', hit Enter, 
click OK/ put whatever instead of 'Enter phrase contents' clisk save **cannot 
save**

What version of the product are you using? On what operating system?
0.82.1

Please provide any additional information below.
autokey-gtk -l 
when I try to save returns:

Traceback (most recent call last):
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 851, in on_save
    persistGlobal = self.__getCurrentPage().save()
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 622, in save
    self.settingsWidget.save()
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 168, in save
    self.abbrDialog.save(self.currentItem)
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/dialogs.py", line 174, in save
    item.set_word_chars(WORD_CHAR_OPTIONS[option])
KeyError: 'Default'
This happens on my Ubuntu 10.10 with LXDE
```

Original issue reported on code.google.com by `pawel.ce...@gmail.com` on 6 Mar 2012 at 12:32

**Comment by GoogleCodeExporter** [comment]
```
I have the same issue on Ubuntu 11.10 and the latest PPA edition.
```

Original comment by `ziegenha...@gmail.com` on 6 Mar 2012 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.82.2
```

Original comment by `cdekter` on 6 Mar 2012 at 11:12
- Changed state: **Fixed**
